README.md

```diff
@@ -1,9 +1,73 @@
-# php-cpu-core-detection
+# PHP CPU Core Count Detection
+
+This PHP script detects the number of CPU cores available on a Linux server. It can be helpful for optimizing performance in multi-threaded applications by dynamically assessing hardware capabilities.
+
+## Purpose
+
+This tool enables developers to retrieve the number of CPU cores on a Linux server using PHP. This is especially useful for performance optimizations and load balancing in server environments.
+
+## Features
+
+- Detects the number of CPU cores on Linux-based systems.
+- Utilizes shell commands (`hash`, `nproc`, `taskset`) to gather system information.
+- Lightweight and easy to integrate into existing PHP applications.
 
 ## Supported OS/Programs
 
-Currently only Linux with `hash`, `nproc` and `taskset` are supported.
+Currently supports **Linux** only, using the following utilities:
+- `hash`
+- `nproc`
+- `taskset`
+
+## Clone this repository:
+   ```bash
+git clone https://github.com/your-repo/php-cpu-core-detection.git
+   ```
+
+## Contributing
+Contributions are welcome! To contribute:
+
+1. Fork this repository.
+2. Make your changes in a new branch.
+3. Submit a pull request with a detailed description of your changes.
+
+## Examples
+
+Check out these examples to see the CPU core detection in action:
+ ### Async
+ ```
```

Comment:
The only thing I could change here is add `php` at the end of this line and the other example start to enable syntax highlighting

Reply:
which line tell me number ill add php in the end